libpod/container.go

```diff
@@ -197,6 +197,8 @@ type ContainerConfig struct {
 	// Information on the image used for the root filesystem/
 	RootfsImageID   string `json:"rootfsImageID,omitempty"`
 	RootfsImageName string `json:"rootfsImageName,omitempty"`
+	// Rootfs to use for the container
```

Comment:
Can you add a note that this conflicts with `RootfsImageID` and `RootfsImageName`?

Reply:
done